grails-app/conf/application.groovy

```diff
@@ -1042,6 +1042,8 @@ license.default = "https://creativecommons.org/licenses/by-nc/3.0/au/"
 projectActivity.notifyOnChange=true
 biocollect.baseURL="https://biocollect.ala.org.au"
 biocollect.projectActivityDataURL="${biocollect.baseURL}/bioActivity/projectRecords"
+biocollect.projectArea.simplificationThreshold=10000
+biocollect.projectArea.simplificationTolerance=0.1
```

Comment:
My understanding is 0.1 degrees is ~10km which is a pretty large tolerance unless we are dealing with very large sites.

Reply:
Updated to be ~10m